netmaster/main.go

```diff
@@ -97,6 +110,51 @@ func execOpts(opts *cliOpts) {
 		log.SetLevel(log.DebugLevel)
 	}
 
+	// Validate listen and control URL options
+	listenURL := strings.Split(opts.listenURL, ":")
+	controlURL := strings.Split(opts.controlURL, ":")
+	if len(listenURL) != 2 {
+		log.Fatalf("Listen URL not in proper format. Valid format: [IP]:Port")
+	}
+	if len(controlURL) != 2 {
+		log.Fatalf("Control URL not in proper format. Valid format: IP:Port")
```

Comment:
Can we also make sure we don't accept 0.0.0.0 for controlURL[0] because without an IP, controlURL does not server its purpose.